ansible/roles/automation/nightly_pr/defaults/main.yml

```diff
@@ -0,0 +1,24 @@
+---
+basedir: /root/openclose_pr
+
+nightly_jobs:
+  - name: testing_master
+    weekdays: "1,3,5"
+    hour: "23"
+    minute: "00"
+  - name: testing_f28
+    weekdays: "2,4"
+    hour: "23"
+    minute: "00"
+  - name: testing_rawhide
+    weekdays: "6"
+    hour: "10"
+    minute: "00"
+  - name: testing_ipa-4.7
+    weekdays: "7"
+    hour: "10"
+    minute: "00"
+  - name: testing_pki
+    weekdays: "7"
+    hour: "20"
+    minute: "00"
```

Comment:
The commit comment is inconsistent with the above definitions:

> Thrusday: f28, template_rawhide, template_pki

The following should be used to match the code:
`Thursday: f28`

Reply:
this items are for running nightly regression flows, not for updating templates.

With this commit we have:

./ansible/roles/automation/nightly_pr/.... --> for running nightly regressions (master, f28, rawhide, etc..)

./ansible/roles/automation/template_pr/.... --> for updating templates (rawhide, pki)

Reply:
My bad, I hadn't understood this!